tests/whitelist.txt

```diff
@@ -4,9 +4,6 @@ integration.fileserver.test_fileclient
 integration.fileserver.test_roots
 integration.loader.test_ext_grains
 integration.loader.test_ext_modules
-integration.loader.test_globals
-integration.loader.test_interfaces
-integration.loader.test_loader
```

Comment:
Yes, that's exactly why. Just mentioned the above in case you still wanted those tests in your names listing.